sdk/include/psdk/msxml2.idl

```diff
@@ -1286,7 +1286,7 @@ interface IXSLProcessor : IDispatch
     HRESULT addParameter(
         [in] BSTR p, 
         [in] VARIANT var, 
-        [in, defaultvalue(L"")] BSTR uri);
+        [in] BSTR uri);
```

Comment:
This is likely to be synced code with Wine. Upstreaming should be done.

Reply:
I added ```#ifndef __REACTOS__```